src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>En tom fil är inte tillåten.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Domänen kunde inte hittas.</target>
```

Comment:
Minor comment: `Domänen` is translated as `domain`. Is it important to maintain here the "host" meaning or it doesn't matter?